routers/api/v1/api.go

```diff
@@ -1460,27 +1455,21 @@ func Routes() *web.Route {
 				m.Combo("/{username}").Get(reqToken(), org.IsMember).
 					Delete(reqToken(), reqOrgOwnership(), org.DeleteMember)
 			})
-			m.Group("/actions", func() {
-				m.Group("/secrets", func() {
-					m.Get("", reqToken(), reqOrgOwnership(), org.ListActionsSecrets)
-					m.Combo("/{secretname}").
-						Put(reqToken(), reqOrgOwnership(), bind(api.CreateOrUpdateSecretOption{}), org.CreateOrUpdateSecret).
-						Delete(reqToken(), reqOrgOwnership(), org.DeleteSecret)
-				})
-
-				m.Group("/variables", func() {
-					m.Get("", reqToken(), reqOrgOwnership(), org.ListVariables)
-					m.Combo("/{variablename}").
-						Get(reqToken(), reqOrgOwnership(), org.GetVariable).
-						Delete(reqToken(), reqOrgOwnership(), org.DeleteVariable).
-						Post(reqToken(), reqOrgOwnership(), bind(api.CreateVariableOption{}), org.CreateVariable).
-						Put(reqToken(), reqOrgOwnership(), bind(api.UpdateVariableOption{}), org.UpdateVariable)
-				})
-
-				m.Group("/runners", func() {
-					m.Get("/registration-token", reqToken(), reqOrgOwnership(), org.GetRegistrationToken)
-				})
-			})
+			actionsGroup(
+				m,
+				reqOrgOwnership(),
+				actionAPI{
+					org.ListActionsSecrets,
+					org.CreateOrUpdateSecret,
+					org.DeleteSecret,
+					org.ListVariables,
+					org.GetVariable,
+					org.DeleteVariable,
+					org.CreateVariable,
+					org.UpdateVariable,
+					org.GetRegistrationToken,
+				},
```

Comment:
I think it should use the field names explicitly. Otherwise, it's difficult to know whether the function really matches the field name: eg: `Struct{Func1, Func2, Func3, Func4}` and `Struct{Func1, Func3, Func2, Func4}` all could compile and run, but mismatched route handlers would cause problems.